extensions/Makefile

```diff
@@ -1,55 +1,55 @@
-# See README for more information about the WebGL extension registry automation.
```

Comment:
Are the changes in this file affecting the line endings? If so please revert them.